protoactor/protos_pb2.py

```diff
@@ -0,0 +1,267 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
I need to investigate a bit on this then. Because it seems weird to include this, since it can be generated any time from `protoc`.